src/nixos-anywhere.sh

```diff
@@ -476,7 +476,9 @@ if [[ ${copy_host_keys-n} == "y" ]]; then
 fi
 nixos-install --no-root-passwd --no-channel-copy --system "$nixos_system"
 if command -v zpool >/dev/null; then
-  zpool export -a || : # we always want to export the zfs pools so people can boot from it without force import
+  # we always want to export the zfs pools so people can boot from it without force import
+  umount -Rv /mnt/
+  zpool export -a || :
```

Comment:
```suggestion
  zpool export -a || true
```

I know it's unchanged, but i find this colon syntax confusing every time i see it :D